core/field_variable.js

```diff
@@ -153,7 +153,7 @@ Blockly.FieldVariable.dropdownCreate = function() {
   }
   options.push([Blockly.Msg.RENAME_VARIABLE, Blockly.RENAME_VARIABLE_ID]);
   options.push([Blockly.Msg.DELETE_VARIABLE.replace('%1', name),
-               Blockly.DELETE_VARIABLE_ID]);
+    Blockly.DELETE_VARIABLE_ID]);
```

Comment:
This should be at least a 4-space indent, and large indentation for syntactic sibling (as it was before) is very common, if not preferred.